akka-cluster-sharding/src/main/scala/akka/cluster/sharding/ShardRegion.scala

```diff
@@ -614,7 +614,7 @@ private[akka] class ShardRegion(
    * oldest node and therefore it is good to send the Register and GracefulShutdownReq to
    * the likely locations of the coordinator.
    */
-  def coordinatorSelection: List[ActorSelection] = {
+  def coordinatorSelection(): List[ActorSelection] = {
```

Comment:
This is side effect free so without parenthesises was fine?

Reply:
Yes, I agree. I added it because it was being used on a method twice ([here](https://github.com/akka/akka/pull/29579/files#diff-9d06486133d0565a63482bccb91579c0L1173-L1174)).

Probably because it looked like a `val`. If it had a `()`, we would probably think twice before calling it like that.

Reply:
Not that it matters that much but, we don't usually use `()` to signal anything but side effects, so a bit weird using it for something else here.